content/en/network_monitoring/devices/syslog.md

```diff
@@ -0,0 +1,198 @@
+   * **`service`**: Optional service name for unified service tagging.
+
+3. Restart the [Docker container][101] to apply the changes.
+
+[101]: /agent/configuration/agent-commands/#restart-the-agent
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Log parsing
+
+The above steps send your Syslog data to Datadog. To ensure NDM correctly associates these logs with your monitored network devices, set up a custom log parsing pipeline that populates the `syslog_ip` tag with each device's source IP address.
+
+**Note**: NDM queries for logs where `source:syslog` **AND** the `syslog_ip` tag matches one of the IPs of the device.
+
+Choose the appropriate scenario based on your network configuration:
+
+#### Scenario A: Direct connection (no proxy)
+
+When network devices send Syslog messages directly to the Datadog Agent, the Agent's `use_sourcehost_tag: true` setting automatically adds a `source_host` tag containing the sender's IP address.
+
+To create the `syslog_ip` tag, remap the `source_host` tag using a Log Processing Pipeline:
+
+1. In Datadog, navigate to **[Logs > Log Configuration > Pipelines][3]**.
+2. Create a pipeline or select an existing one.
+3. Add a **Log Remapper** processor with the following configuration:
+   * **Source of the tag:** `source_host`
+   * **Target tag:** `syslog_ip`
+
+#### Scenario B: Using a Syslog proxy
+
+When network devices send Syslog through a proxy (such as rsyslog or syslog-ng), the `source_host` tag reflects the proxy's IP address instead of the original device's IP.
+
+To create the `syslog_ip` tag:
+
+1. **Configure the proxy** to include the original source IP in the Syslog message payload.
+   * **rsyslog example:** Use a template like `$template CustomFormat,"%fromhost-ip% %msg%\n"` to prepend the source IP to each message.
+2. **Create a Log Processing Pipeline** to extract and map the IP address:
+   1. In Datadog, navigate to [**Logs > Log Configuration > Pipelines**][3].
+   2. Create a pipeline or select an existing one.
+   3. Add a **Grok Parser** processor to extract the IP address from the message into a temporary attribute (for example, `@temp_ip`).
+   4. Add a **Log Remapper** processor with the following configuration:
+      * **Source of the attribute:** `@temp_ip` (or your chosen attribute name)
+      * **Target tag:** `syslog_ip`
+      * **Preserve source attribute:** Uncheck this option to remove the temporary attribute
+
+For detailed instructions, see the [Datadog Log Pipelines documentation][2].
+
+## Verify Syslog messages
+
+After your network devices are configured and the Datadog Agent is running, you can verify that Syslog data is being collected and sent to Datadog:
+
+1. Navigate to the **[Log Explorer][4]** in your Datadog account.
+2. **Filter by `source:syslog`** (or whatever source you specified in your `conf.yaml` file).
+3. You should see your network device Syslog messages appearing in the Log Explorer.
```

Comment:
I would incorporate this into the previous step; it's not really an action, but a consequence of the previous action, so I don't think it needs to be its own step.